src/coreclr/vm/eventing/userevents/LinuxTracepoints/README

```diff
@@ -0,0 +1 @@
+This code is copied from https://github.com/microsoft/LinuxTracepoints/
```

Comment:
If this is a copy and one that we intend to follow in an upstream manner, we should consider placing it under https://github.com/dotnet/runtime/tree/main/src/native/external

Reply:
Moved to src/native/external